django-verdant/rcasitemaps/views.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import absolute_import, unicode_literals
```

Comment:
So this views file is needed because its an older version of Wagtail?

Reply:
That's right, the [older sitemap version](https://github.com/wagtail/wagtail/blob/v1.6.3/wagtail/contrib/wagtailsitemaps/views.py) caused a few errors, mainly no `protocol` so to save time i'm using the newer view